### DISCUSSION TITLE
How to use special inputs fields ?

### DISCUSSION THREAD

**Discussion by StevenRiot** [duncanmcclean/guest-entries#70] [Help] [answered]
Hey Duncan,
I've read the documentation but I'm not sure how to use those special input fields (slug/date/published)

Could you please provide a quick example ?
Many thanks in advance

**Comment by duncanmcclean** [comment, marked as answer]
Although those fields are treated specially within Statamic, you can pretty much treat them like normal fields inside a guest entry form.

For example:

```antlers
{{ guest-entries:create collection="blog" }}
    <input type="text" name="slug">
    <input type="date" name="date">
    <input type="checkbox" name="published">
{{ /guest-entries:create }}
```
